cranelift/codegen/src/verifier/mod.rs

```diff
@@ -1504,7 +1487,9 @@ impl<'a> Verifier<'a> {
             ));
         }
         for (arg, param) in args.zip(block_params.iter()) {
-            let arg_ty = self.block_call_arg_ty(arg, inst, target_type, errors)?;
+            let Some(arg_ty) = self.block_call_arg_ty(arg, inst, target_type, errors)? else {
+                return Ok(());
```

Comment:
Perhaps we could `continue` here so we still check the remaining args?

Reply:
Done